tests/integration/test_analysisd/test_signature_id/data/rules_samples/null_signature_id_rules.xml

```diff
@@ -0,0 +1,8 @@
+
+<group name="syscollector,">
+
+    <rule id="999998" level="3" >
+        <if_sid>999999</if_sid>
```

Comment:
Yes, this represents a null if_sid, because the rule referenced does not exist, so it is ignored, leaving a null field... If you place the tag but have no value inside it actually invalid.